src/specklepy/objects/structural/results.py

```diff
@@ -21,28 +21,28 @@ class ResultSet1D(Result, speckle_type=STRUCTURAL_RESULTS + "ResultSet1D"):
 
 class Result1D(Result, speckle_type=STRUCTURAL_RESULTS + "Result1D"):
     element: Element1D = None
-    position: float = 0.0
-    dispX: float = 0.0
-    dispY: float = 0.0
-    dispZ: float = 0.0
-    rotXX: float = 0.0
-    rotYY: float = 0.0
-    rotZZ: float = 0.0
-    forceX: float = 0.0
-    forceY: float = 0.0
-    forceZ: float = 0.0
-    momentXX: float = 0.0
-    momentYY: float = 0.0
-    momentZZ: float = 0.0
-    axialStress: float = 0.0
-    shearStressY: float = 0.0
-    shearStressZ: float = 0.0
-    bendingStressYPos: float = 0.0
-    bendingStressYNeg: float = 0.0
-    bendingStressZPos: float = 0.0
-    bendingStressZNeg: float = 0.0
-    combinedStressMax: float = 0.0
-    combinedStressMin: float = 0.0
+    position: float = None
```

Comment:
In every place, where you say 
`{attribute_name}: float = None`
The actual definition should be `{attribute_name}: Optional[float] = None

If a `None | float` value is declared as just a `float` type, IDE-s and type checkers do not require you to handle the `None` case, which will most likely cause bugs. To be explicitly safe I strongly prefer using `Optional[float]` for these type definitions.

Please refactor them to that

Reply:
@gjedlicska Good one, I updated the type definition now.